pkg/scheduler/framework/plugins/noderesources/least_allocated_test.go

```diff
@@ -126,11 +64,15 @@ func TestNodeResourcesLeastAllocated(t *testing.T) {
 			// CPU Score: ((4000 - 0) * MaxNodeScore) / 4000 = MaxNodeScore
 			// Memory Score: ((10000 - 0) * MaxNodeScore) / 10000 = MaxNodeScore
 			// Node2 Score: (MaxNodeScore + MaxNodeScore) / 2 = MaxNodeScore
-			pod:          &v1.Pod{Spec: noResources},
-			nodes:        []*v1.Node{makeNode("machine1", 4000, 10000), makeNode("machine2", 4000, 10000)},
-			args:         config.NodeResourcesLeastAllocatedArgs{Resources: defaultResourceLeastAllocatedSet},
-			expectedList: []framework.NodeScore{{Name: "machine1", Score: framework.MaxNodeScore}, {Name: "machine2", Score: framework.MaxNodeScore}},
 			name:         "nothing scheduled, nothing requested",
+			requestedPod: &noResourcePod,
+			nodes: []*v1.Node{
+				&st.MakeNode().Name("node1").Capacity(map[v1.ResourceName]string{"cpu": "4000", "memory": "10000"}).Node,
```

Comment:
It's conventional to get the obj by `st.MakeXYZ().....Obj()`. (applies elsewhere)

Reply:
done:)